### DISCUSSION TITLE
游戏bug吗

### DISCUSSION THREAD

**Discussion by HuangTangyan** [Meloong-Git/PCL#1994] [提问] [answered]
请问在我的世界中物品栏一旦有东西就会花屏怎么办https://github.com/Hex-Dragon/PCL2/assets/135442404/75c6c03d-6a49-4faf-8091-652ad75eb7a4

**Comment by HuangTangyan** [comment]
https://github.com/Hex-Dragon/PCL2/assets/135442404/6d80f401-ff5d-49d1-bac4-c1c52cedfc93

**Comment by WForst-Breeze** [comment, marked as answer]
请尝试如下方案：
1. 打开任务管理器，找到 Minecraft 的进程，打开文件所在位置
![image](https://github.com/Hex-Dragon/PCL2/assets/110760354/af44c116-6796-440c-9f4e-1525a703bd17)
2. 复制路径，前往 Windows 系统的 **设置-系统-屏幕-图像设置-图形性能首选项-浏览**
3. 粘贴复制的路径，找到 javaw，单击添加
4. 单击 **选项-高性能**
![image](https://github.com/Hex-Dragon/PCL2/assets/110760354/0ab9c46f-b9e1-4d42-94d8-48477f81ed61)
5. 重启游戏

如仍无法解决问题，请在下方补充~
